source/production/F0.Minesweeper.Logic/Mineplacer/RandomMineplacer.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using F0.Minesweeper.Logic.Abstractions;
+
+namespace F0.Minesweeper.Logic.Mineplacer
+{
+	internal class RandomMineplacer : IMineplacer
+	{
+		IEnumerable<ILocation> IMineplacer.PlaceMines(IEnumerable<ILocation> possibleLocations, uint mineCount, ILocation clickedLocation)
+			=> possibleLocations
+				.OrderBy(_ => Guid.NewGuid())
```

Comment:
OK - so here is a "small" one: how do we test Random.
... long story short ... as we discussed this in length yesterday:
Perhaps have a generic "Shuffle & Take" interface, where the `Fake...`.Implementation takes a Location-Collection which is than returned in that order, and the implementation for the production code does the "OrderBy & Take" logic.
And for the tests with the `Fake...` we could have a c'tor or some nice method taking C# tuples (ValueTuple) to have a readable syntax like `(1, 1), (2, 2), (3, 3)` which would deterministically return the top left, middle, and bottom right location of a _3x3_ field.

Reply:
Opened #24